chsdi/models/vector/bafu.py

```diff
@@ -12,6 +12,26 @@
 Base = bases['bafu']
 
 
+class AlpweidenHerdenschutzhunde(Base, Vector):
+    __tablename__ = 'alpenweiden_herdenschutzhunde'
+    __table_args__ = ({'schema': 'fauna', 'autoload': False})
+    __bodId__ = 'ch.bafu.alpweiden-herdenschutzhunde'
+    __queryable_attributes__ = ['name', 'kotname', 'kottel', 'hotemail', 'code_refverhalten', 'code_hundepraesenz', 'code_hinweis']
```

Comment:
* ``kotname`` does not exist in ``bafu_master.fauna.alpenweiden_herdenschutzhunde``, there is a column called ``kontname`` which is empty??
* ``kottel`` same as ``kotname``
* ``hotemail`` does not exist neither there is a column ``kontemail``

please fix the typos names of the columns

* ``code_refverhalten``, ``code_hundepraesenz``, ``code_hinweis`` are coded values, i would recommend to not implement attribute search on such attributes, furthermore 102 of 103 objects have the same value, in my eyes it's nonsense to have a attribute search here, @mariokeusen your opinion please?